dlang.org.ddoc

```diff
@@ -242,16 +242,16 @@ $(SUBMENU
 )
 NAVIGATION_RESOURCES=
 $(SUBMENU
+    $(ROOT_DIR)acknowledgements.html, Acknowledgments,
+    https://wiki.dlang.org/Books, Books,
     $(ROOT_DIR)tools.html, D-Specific Tools,
```

Comment:
> That page needs to be dealt with, I'm just not clear on how.

Maybe we should join it with Editors and IDEs (and build tools and debuggers etc). Would add some meat to the page, and slim down the menu. Having links to the Wiki directly in the Menu is a bit confusing anyway.